db/migrate/20240422064305_migrate_product_supplier.rb

```diff
@@ -0,0 +1,11 @@
+class MigrateProductSupplier < ActiveRecord::Migration[7.0]
+  def up
+    ActiveRecord::Base.connection.execute(<<-SQL
+      UPDATE spree_variants
+      SET supplier_id = spree_products.supplier_id
+      FROM spree_products
+      WHERE spree_variants.product_id = spree_products.id
```

Comment:
This looks right, but  I have to confess I'm not very familiar with the UPDATE syntax.
So I thought it would be good to have a spec for this (we have some migration specs already), but it's a pretty simple statement so I will just go and look up the syntax ;)

Reply:
Ok I looked it up, that checks out 👍